flogin/settings.py

```
@@ -3,6 +3,8 @@
 import logging
 from typing import TYPE_CHECKING, Any, overload
 
+from .enums import SettingTemplateInputType
```

Ruff (0.8.2)

`.enums.SettingTemplateInputType` imported but unused

Remove unused import: `.enums.SettingTemplateInputType`

(F401)